### ISSUE TITLE
[11/12 regression] error: 'fenv_t' has not been declared in '::' -- cross toolchain fails to build

### ISSUE THREAD

**Issue by oferShinaar** [riscv/riscv-gcc#283]
There is a bug in cross-building 11.1.
[GCC Bugzilla - Bug 100017](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100017)

The target fix is 11.2. Do you have a local fix?

Thanks,
Ofer

**Comment by kito-cheng** [comment]
No, I would suggest wait Jonathan Wakely to post his patch, maybe you could ping him on bugzilla?

**Comment by oferShinaar** [comment]
seems it was targeted to gcc 11.3
